Kernel/Arch/aarch64/Exceptions.cpp

```diff
@@ -74,7 +74,10 @@ static void setup_el1()
     system_control_register_el1.UMA = 1;  // Don't trap access to DAIF (debugging) flags of EFLAGS register
     system_control_register_el1.SA0 = 1;  // Enable stack access alignment check for EL0
     system_control_register_el1.SA = 1;   // Enable stack access alignment check for EL1
-    system_control_register_el1.A = 1;    // Enable memory access alignment check
+
+    // NOTE: Disable memory access alignment check as we cannot enforce
+    //       proper alignment for userspace applications.
+    system_control_register_el1.A = 0;
```

Comment:
Why can't we require userspace apps to only do aligned memory accesses?

Do the currently-loaded first-party userspace apps do unaligned reads? Where?

Reply:
We could require that by adding the same flags as we do for the kernel binary (`-mstrict-align -Wno-cast-align`), however that means we also need to add those flags for all the ports. I thought it was better if we don't enforce that for userspace applications. And it's not possible to only allow unaligned accesses for EL0, so we need to change that in SCTLR_EL1.

It happens all over the place in our libraries, I don't remember where exactly I hit it tho.

Reply:
I'm fine with either approach, but since we discussed and enabled the enforcement of aligned memory accesses for the kernel, I think it's okay to do the same for userspace now. So I went ahead and removed the previous commit that would disable the check, and instead added the `-mstrict-align -Wno-cast-align` flags to `serenity_compile_options.cmake` in 86b0978dd07653cff761ec57ab25f5985cc7ab8f